config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  devise_for :users
+
+  unauthenticated :user do
+    root to: 'users#index'
+  end
+
+  authenticated :user do
+    root 'groups#index', as: :authenticated_root
+  end
```

Comment:
- [ ] Dear @Nasirkhan294, As mentioned by the previouse reviewers, Please implement a Splash screen and set it as a root route. You have a `root route` for `unauthenticated` and `authenticated` users. but in both cases, you have set the root route to something other than the splash page. Please update your `root route` configuration to redirect you to the splash page. I would suggest setting the `unauthenticated` user `root route` to the `splash page/home page` instead of the `users#index`.

> Keep in mind the `Devise` will redirect the `users#index` to the login page as long as the user is not signed in, hence there is no way we can see the `splash` page.

Reply:
Waiting for your reply @TadesseDev

Reply:
- [ ] @Nasirkhan294 What @TadesseDev is trying to explain is that currently, the user can`t access the splash page as by default devise will redirect the user to the login page even when trying to edit the route manually, it will redirect the user to the login page, the best solution is to create a controller for the splash screen then add a view for it and set it as the root route, using users for the route will always be overridden by devise.

Reply:
This change has been done well

Reply:
> Waiting for your reply @TadesseDev

Sorry for the late replay @Nasirkhan294, I see you got it fixed. Good job ✅